.vib/reportserver/goss/vars.yaml

```diff
@@ -0,0 +1,22 @@
+binaries:
+  - gosu
+  - java
+  - mysql
+  - render-template
+files:
+  - paths:
+      - /opt/bitnami/reportserver/.componentsSkip
+      - /opt/bitnami/reportserver/WEB-INF/classes/persistence.properties
+      - /opt/bitnami/reportserver/WEB-INF/classes/reportserver.properties
+      - /opt/bitnami/reportserver/WEB-INF/classes/rsinit.properties
+directories:
+  - mode: "0775"
+    paths:
+      - /bitnami/tomcat/webapps
+      - /opt/bitnami/reportserver
+root_dir: /opt/bitnami
+webapp_folders:
+  - /bitnami/tomcat/host-manager
```

Comment:
This should be `/bitnami/tomcat/webapps/host-manager` right?

Reply:
Yes